### PR TITLE
Ma/jwks cache

### PR DESCRIPTION
**Pull request by markusahlstrand**
We were using the wrong constant for the cache control header. Also decreased it to 5 minutes.

### CODE REVIEW

File: src/hooks/update-client.ts [hunk 1]
```diff
@@ -90,7 +90,7 @@ export async function updateClientInKV(env: Env, applicationId: string) {
       sender_email: tenant.sender_email,
       sender_name: tenant.sender_name,
       language: tenant.language,
-      suppport_url: tenant.support_url,
+      support_url: tenant.support_url,
```
Comment (markusahlstrand) [inline src/hooks/update-client.ts:93]: This is merged in another PR already.. Github isn't what it used to be :)

File: src/routes/tsoa/jwks.ts [hunk 2]
```diff
@@ -54,7 +55,9 @@ export class JWKSRoutes extends Controller {
     this.setHeader(headers.accessControlAllowMethod, "GET");
     this.setHeader(
       headers.cacheControl,
-      `public, max-age=${CERTIFICATE_EXPIRE_IN_SECONDS}, stale-while-revalidate=${CERTIFICATE_EXPIRE_IN_SECONDS}, stale-if-error=86400`,
+      `public, max-age=${JWKS_CACHE_TIMEOUT_IN_SECONDS}, stale-while-revalidate=${
+        JWKS_CACHE_TIMEOUT_IN_SECONDS * 2
+      }, stale-if-error=86400`,
```
Comment (danielgent) [inline src/routes/tsoa/jwks.ts:60]: LOL. Nice spot

**Review by danielgent — APPROVED** [review]
The stuff of nightmares!

File: src/hooks/update-client.ts (hunk 1 above)
Comment (danielgent) [reply src/hooks/update-client.ts]: I totally feel this. I get these kind of issues *all the time*

**Pull request merged**